Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -26,6 +26,9 @@ Neos:
 
       # Defines the PSR-15 middlewares that a request goes through in a flow
       middlewares:
+        'session':
```

Comment:
shouldn't that be placed much further back in the middleware chain?
It used to be in the `postprocess` chain before so it should probably be put after the `dispatch` middleware at least

Reply:
Ah, nevermind.. I assume it's the other way around now :)

Reply:
It "wraps" the request in the implementation. Maybe we need to split it to two parts again?

Reply:
I'm not sure about the best practices here but my feeling is that excessive nesting will make it harder to debug since it basically makes the stack trace of every call between one step longer

Reply:
That's indeed a drawback of the middlewares/onion approach, so let's keep those merged.